dll/apisets/api-ms-win-core-crt-l1-1-0.spec

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by update.py
 
 @ stdcall __isascii() msvcrt.__isascii
-@ stdcall _atoi64() msvcrt._atoi64
+@ cdecl -ret64 _atoi64() msvcrt._atoi64
```

Comment:
This does not seem useful for forwarders. Do you expect this to achieve anything?

Reply:
At least, code consistency with ReactOS/Wine and `cdecl`/`stub` cases.
If you confirm `-ret64` is useless and unwanted for forwarders, I'll remove them.

Reply:
A forwarder does not contain any code, it is simply a string in the dll specifying the target dll + function name

Reply:
Removed.